README.md

```diff
@@ -11,24 +17,32 @@ pip install ollama
 ## Usage
 
 ```python
-import ollama
-response = ollama.chat(model='llama3.1', messages=[
+from ollama import chat
+from ollama._types import ChatResponse
```

Comment:
`ollama._types` should not be imported directly. `ChatResponse` is exported explicitly by `ollama`. Also why the change to `from ollama import ...`?

```suggestion
from ollama import chat, ChatResponse
```

Reply:
Oh you're right let me fix